.github/workflows/linux.yml

```diff
@@ -165,21 +165,21 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: ./scripts/install_zig.sh
-      - run: zig/zig build fuzz_ewah -- --seed 123
-      - run: zig/zig build fuzz_lsm_segmented_array -- --seed 123
-      - run: zig/zig build fuzz_lsm_tree -- --seed 123 --events-max 400
-      - run: zig/zig build fuzz_vsr_journal_format -- --seed 123
-      - run: zig/zig build fuzz_vsr_superblock -- --seed 123 --events-max 3
-      - run: zig/zig build fuzz_vsr_superblock_quorums -- --seed 123
-
-      # Build the fuzzers, but don't run them, since they're slow.
-      - run: zig/zig build build_fuzz_lsm_manifest_log
-      - run: zig/zig build build_fuzz_vsr_free_set
-      - run: zig/zig build build_fuzz_lsm_forest
+      - run: zig/zig build fuzz -- ewah                   --seed=123
```

Comment:
Thinking about it; the point of these is to do a smoke test on all of our fuzzers to make sure they build and at least pass at a very basic level. 

Given that, would be handy if we could just do `zig/zig build fuzz -- all --seed=123 --events-max=10` here and keep our yaml O(1) :wink:.

Reply:
Most excellent suggestion! It also allows us to take advantave of exhaustiveness checking, which immediately points out that there's also `cache_map` fuzz, which is conspiciously absent form our CI setup! Will run that in a follow up!